simplq/src/styles/adminPage.module.scss

```diff
@@ -29,8 +33,9 @@
       flex-direction: row;
       align-items: center;
       .expand-icon {
-        margin-left: 2rem;
+        margin-left: 7rem;
```

Comment:
Why did we add this margin?

Reply:
As per the suggested layout in the [issue](https://github.com/SimplQ/simplQ-frontend/issues/325), the chevron is placed to the right of the div.
As the Previous layout was a little bit zoomed in the margin of 2rem's was sufficient to place the chevron/ expand icon to the right.

If we did not add the extra margin, it would look like the image below
![chev-without-margin](https://user-images.githubusercontent.com/11575314/95146247-f44d5980-079a-11eb-98bc-3d208ad2b7e2.png)

Reply:
I don't think this is the right solution. This has been solved independently in another PR. Can you pull the latest changes?